.github/workflows/test.yml

```diff
@@ -13,7 +13,6 @@ jobs:
           - windows-latest
           - macos-latest
         go:
-          - '1.9'
```

Comment:
GRPC test couldn't pass when Go v1.9 because `crypt.Certificate` doesn't have `urls` at this version.
https://github.com/aws/aws-xray-sdk-go/pull/264/checks?check_run_id=1679759717

Do you have plan to drop old version supports?
If we drop dep support ( #266 ), v1.10 will be also dropped.